widget/contrib/wifi.lua

```diff
@@ -0,0 +1,70 @@
+local awful     = require("awful")
+local wibox     = require("wibox")
+local helpers   = require("lain.helpers")
+
+function factory(args)
+    local wifi      =  { widget = wibox.widget.textbox() }
+    local args      = args or {}
+    local timeout   = args.timeout or 2
+    local settings  = args.settings or function() end
+         
+    local command = {
```

Comment:
Don't use `ping` to check connection. `nmcli` must have its own method. Maybe `nmcli general`.